### ISSUE TITLE
Provide custom slug URL for CRADLE studies

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#794]
# TL;DR

Create a link directly to the study list view that is pre-filtered for CRADLE studies

# Narrative
As a researcher, I want to be able to send out a simple/memorable link to a view of all studies set as CRADLE

<details>
<summary>Feature Request fields</summary>
<p>
**With this `details` wrapper, you can just use this ticket to record your findings.
When you're ready to turn this into an implementation request, delete the scoping label and
this wrapper, leaving only the filled out information.**

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
Leave important notes/suggestions here for implementation (important files, suggested approaches, etc.)

</p>
</details>

**Comment by msheskin** [comment]
[Not sure the best way to comment/discuss, let me know if dropping thoughts here is not what I should do]: It would be amazing if we could do some customization of this page, rather than it being a "search results" page that looks like every other one. At a minimum, a tiny bit of "standardized branding" of the sort seen with ECCl logo on this CHS page: https://childrenhelpingscience.com/eccl-mit -- fancier would be even better (e.g., showing different things to different families based on some conditional logic, such as if the number of cradle studies they have done is zero, they see something different compared to if it is 1+

**Comment by mekline** [comment]
Closing - for phase 1, this is accomplished with an enriched CRADLE lab page (see issue #918)
